connector/mqtt/vernemq/v2k-brigde/README.md

```diff
@@ -18,16 +18,13 @@ HOSTNAME                 | Hostname to be used in the certificate common name
 EJBCA_HOSTNAME           | Address of the EJBCA broker                                         | localhost           | hostname/IP    |
 EJBCA_PORT               | Port of the EJBCA broker                                            | 5583                | integer        |
 AUTH_URL                 | Address of the auth service                                         | http://auth:5000    | hostname/IP    |
```

Comment:
I think we can change from `AUTH_URL` to `AUTH_ADDRESS`, and from `http://auth:5000` to `auth:5000`